src/lints/unwind_function_missing.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "unwind_function_missing",
+    human_readable_name: "unwind function missing",
+    description: "pub fn changed from an unwind-capable ABI to the same-named ABI without unwind: \"C-unwind\" to \"C\"",
```

Comment:
We don't necessarily know that the change was *specifically* `C-unwind` to `C`. It's possible that the change is from a different unwind-capable ABI to its non-unwind-capable equivalent.

Reply:
I see, I will change the description to "pub fn changed from an unwind-capable ABI to the same-named ABI without unwind" then. So that the description doesn't specify if the change was *specifically* `C-unwind` to `C` to anticipate new types of unwind capable ABI being added to rust in the future. Is this an appropriate approach to accommodate this issue?

Reply:
Sounds good to me!

Here are a few more unwind-capable ABIs in case you're curious: https://github.com/rust-lang/rfcs/blob/master/text/2945-c-unwind-abi.md#other-unwind-abi-strings

If we expand the set to not-yet-stable ABIs, the set is even bigger:
https://doc.rust-lang.org/beta/unstable-book/language-features/c-unwind.html

We might want to use the RFC that introduced `unwind` ABIs as the reference link on line 6 btw: https://github.com/rust-lang/rfcs/blob/master/text/2945-c-unwind-abi.md